src/frontends/paddle/tests/read_paddle_model_test.cpp

```diff
@@ -62,8 +62,16 @@ TEST(Paddle_Reader_Tests, LoadModelMemoryToCore) {
     const auto relu = std::make_shared<ov::opset1::Relu>(conv2d->output(0));
     relu->set_friendly_name("relu_0.tmp_0");
     relu->output(0).get_tensor().add_names({"relu_0.tmp_0"});
-    const auto result = std::make_shared<ov::opset1::Result>(relu->output(0));
-    result->set_friendly_name("relu_0.tmp_0/Result");
+    const auto bias = std::make_shared<ov::opset1::Constant>(ov::element::f32, ov::Shape{}, 0.0);
```

Comment:
Does it still work in older paddle version? If not, I think these two test cases needn't to be compatible with old paddle version. I recommend add a flag in cmake to indicate the paddle version and these two test cases only enabled in latest paddle version we support

Reply:
> Does it still work in older paddle version? If not, I think these two test cases needn't to be compatible with old paddle version. I recommend add a flag in cmake to indicate the paddle version and these two test cases only enabled in latest paddle version we support

it can passed.